pkg/operator/http_test.go

```diff
@@ -118,7 +118,7 @@ func TestAPIV1ReportsGet(t *testing.T) {
 	}{
 		"report-finished-no-results": {
 			reportName: testReportName,
-			report:     testhelpers.NewReport(testReportName, namespace, testQueryName, reportStart, reportEnd, metering.ReportStatus{}, nil, false, nil),
+			report:     testhelpers.NewReport(testReportName, namespace, testQueryName, nil, reportStart, reportEnd, metering.ReportStatus{}, nil, false, nil),
```

Comment:
Why the nil before reportStart parameter?

Reply:
Looks like reportStart is the first key in the TestAPIV1ReportsGet struct.

Reply:
This PR adds `inputs` in that slot.